tests/scripts/benchmark_nnpkg.sh

```diff
@@ -128,7 +120,7 @@ for i in "${model_lists[@]}"; do
 
   echo "" >> ${outfile}
 
-  TFLITE_CMD="LD_LIBRARY_PATH=./Product/out/lib THREAD=3 ${tflite_run} -r 10 -m 1 -p 1"
+  TFLITE_CMD="THREAD=3 ${tflite_run} -r 10 -m 1 -p 1"
```

Comment:
Is `LD_LIBRARY_PATH` not necessary any longer?

Reply:
Right.

https://github.com/Samsung/ONE/blob/497fcd27fbd9cd935296e913664a09f32cbfbb62/infra/nnfw/CMakeLists.txt#L9-L10

`CMAKE_INSTALL_RPATH` helps to find `*.so` libraries.

Reply:
I confimed `LD_LIBRARY_PATH` was not necessary before this PR though it is removed by this PR.